src/test/replaygaintest.cpp

```diff
@@ -5,6 +5,10 @@
 #include "test/mockedenginebackendtest.h"
 #include "track/replaygain.h"
 
+// The gtest `ASSERT_*` macros return a void expression and thus may lead to a
+// lot of clazy warnings.
+// clazy:excludeall=returning-void-expression,missing-qobject-macro
```

Comment:
why `missing-qobject-macro`?

Reply:
We have some test classes that inherit from it. But adding Q_OBJECT won't help, because gtest generates new classes from it via macro magic, and we can't add Q_OBJECT to these anyway.

Reply:
makes sense, this should also go in the comment imo.